src/CodeStyle/Core/Analyzers/Microsoft.CodeAnalysis.CodeStyle.csproj

```diff
@@ -26,7 +26,6 @@
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.CodeStyle" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.CodeStyle.Fixes" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CodeStyle.UnitTestUtilities" />
-    <InternalsVisibleTo Include="Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.VisualBasic.CodeStyle.UnitTests" />
```

Comment:
`Microsoft.CodeAnalysis.VisualBasic.CodeStyle.UnitTests` has tests?

Reply:
Project is completely deleted. I'll clean this up.